lib/types/uuid.js

```diff
@@ -72,20 +72,14 @@ class Uuid {
      * @returns {Uuid}
      */
     static random(callback) {
-        function getRandomBytes(cb) {
-            return crypto.randomBytes(16, cb);
-        }
         if (callback) {
-            getRandomBytes(function (err, buffer) {
-                if (err) {
-                    return callback(err);
-                }
-                return callback(null, createUuidFromBuffer(buffer));
-            });
-        } else {
-            const buffer = getRandomBytes();
-            return createUuidFromBuffer(buffer);
+            try {
+                return callback(null, new Uuid(rust.getRandomUuidV4()));
+            } catch (err) {
+                return callback(err);
+            }
         }
```

Comment:
:wrench: Please write a comment that you are not sure what error could be thrown here, and you add it just in case.